### ISSUE TITLE
Installing blog breaks admin page after the latest update grav-RC5 release

### ISSUE THREAD

**Issue by danreb** [giansi/grav-plugin-blog#2]
Whoops \ Exception \ ErrorException (E_ERROR)

Call to a member function url() on a non-object 

 Open: /home/gravdemo/public_html/gallery/user/plugins/blog/blog.php

```
        $this->grav['assets']->add(sprintf('plugin://blog/css/%s_blog.css', $framework));
        if ($this->config->get('plugins.blog.add_framework_assets')) {
            $method = 'add' . ucfirst($framework);
            $this->$method();
        }

        $twig = $this->grav['twig'];
        $feedUrl = $baseUrl = $this->grav['page']->parent()->url();
        if ($baseUrl == '/') {
            $baseUrl = '';
```

**Comment by ghost** [comment]
Ditto.

**Comment by giansi** [comment]
@danreb @coelmay could you try grabbing the master branch and let me know if that fixes?

**Comment by ghost** [comment]
@giansi Put master branch in plugins directory. Didn't cause any exceptions.

However, somehow it breaks the **back** button at top of page editor, point to `/pages` rather than `/<install-directory>/pages`—though why you modify this I am not sure :-/

And another thing, should be breadcrumbs be above or below the content? Currently they are below.

**Comment by giansi** [comment]
@coelmay thank you for your feedback. I can't find where the back button breaks: could you help me to find the problem adding a screenshot? Could you tell me which theme are you using? Where are breadcrumbs below the content?

**Comment by ghost** [comment]
I am using the default Antimatter theme.

You can see blog page and test post at http://fukr.co/grav-admin/blog (better than screenshots)—breadcrumbs below.

Not sure how the back button breaks either. grep'd the plugin files recursively, couldn't find anything.

Two screenshots,

with plugin enabled:
<img width="1280" alt="grav-blog-enabled" src="https://cloud.githubusercontent.com/assets/14990463/11358675/c39c7db4-926c-11e5-880d-4e2d73f8f6cd.png">

with plugin disabled:
<img width="1280" alt="grav-blog-disabled" src="https://cloud.githubusercontent.com/assets/14990463/11358701/21835998-926d-11e5-9941-5824c9ac4ddb.png">

**Comment by giansi** [comment]
Ok, got them. I will fix everything today

**Comment by giansi** [comment]
@danreb @coelmay everything should work now: could you try again from master branch?

**Comment by giansi** [comment]
Fixed by 1.1.1 release. Thank you
